Task1 2023/HRLibrary.UnitTests/Vector2DUnitTests.cs

```diff
@@ -0,0 +1,62 @@
+﻿
+using System;
+using NUnit.Framework;
+using Task1_2023;
+
+namespace Task1_2023.Vector2DUnitTest
+{
+    [TestFixture]
+    public class Vector2DTests
+    {
+        private Vector2D CreateVector2D()
+        {
+            return new Vector2D(10, 15.5);
+        }
+
+        [Test]
+        public void ConstructorTest()
+        {
+            var vector = CreateVector2D();
+
+            Assert.That(vector.X, Is.EqualTo(10));
+            Assert.That(vector.Y, Is.EqualTo(15.5));
+            Assert.That(vector.Length, Is.EqualTo(Math.Sqrt(10 * 10 + 15.5 * 15.5)));
+        }
+        
+        [Test]
+        public void ToStringTest()
+        {
+            var vector = CreateVector2D();
+
+            Assert.That(vector.ToString(), Is.EqualTo("(10,15.5)"));
```

Comment:
Тест не проходит. Результат работы вашей перегрузки `ToString()` зависит от региональной настройки ОС. Всё дело в десятичном разделителе (точка или запятая).
Надо сделать независимым от этого либо `ToString()`, либо тест, например, так
```suggestion
            Assert.That(vector.ToString(), Is.EqualTo("(10," + (15.5).ToString() +")"));
```